colander/__init__.py

```diff
@@ -2273,6 +2270,21 @@ def __init__(self, *args, **kw):
             raise Invalid(self,
                           'Sequence schemas must have exactly one child node')
 
+    def clone(self):
+        """ Clone the schema node and return the clone.  All subnodes
+        are also cloned recursively.  Attributes present in node
+        dictionaries are preserved."""
+
+        # Cloning a ``SequenceSchema`` doesn't work with ``_SchemaNode.clone``.
+
+        children = [node.clone() for node in self.children]
+        cloned = self.__class__(self.typ, *children)
+
+        attributes = self.__dict__.copy()
+        attributes.pop('children', None)
+        cloned.__dict__.update(attributes)
+        return cloned
```

Comment:
ISTM we need one or more separate tests for the overridden `SequenceNode.clone`.